tutorials/conf.py

```diff
@@ -373,10 +374,12 @@
 # Note: links to Python doc only work if you are online or have python.inv
 #     file. To download it, run:
 #     curl https://docs.python.org/3/objects.inv >python.inv
-intersphinx_mapping = {
-        'python-openflow': ('https://docs.kytos.io/python-openflow', None),
-        'kytos': ('https://docs.kytos.io/kytos', None),
-        'python': ('https://docs.python.org/3', None)}
+# intersphinx_mapping = {
+#         'python-openflow': ('https://docs.kytos.io/python-openflow', None),
+#         'kytos': ('https://docs.kytos.io/kytos', None),
+#         'python': ('https://docs.python.org/3', None)}
+
+intersphinx_mapping = {}
```

Comment:
Is this tested? We really don't need the intersphinx_mapping?

Reply:
Probably I made these changes in first time I tested this repo. Sorry @hdiogenes, on the next commit I undo these changes.